src/extensions/averagewaveforms2/AverageWaveformsView/electrodesLayer.ts

```diff
@@ -0,0 +1,158 @@
+import { CanvasPainter } from '../../CanvasWidget/CanvasPainter';
```

Comment:
This electrodes layer was mostly copied from elec geom view

Reply:
My plan is to share this code between the widgets. However, I needed to first allow different props types for different layers in the same canvaswidget. So that's why it exists in two places for now. Sorry I don't have a line-by-line difference.